pkg/yurthub/certificate/hubself/cert_mgr.go

```diff
@@ -85,6 +86,26 @@ type yurtHubCertManager struct {
 	stopCh                chan struct{}
 }
 
+// GetYurthubConfRootDir
+func GetYurthubConfRootDir() string {
```

Comment:
how about change `hubRootDir` to `HubRootDir`? so yurtctl can use `HubRootDir` directly.  and `GetYurthubConfRootDir ` func is not need to add.

Reply:
In NewYurtHubCertManager, rootDir is got in the same way, from config first.
```
        hn := projectinfo.GetHubName()
        if len(hn) == 0 {
                hn = hubName
        }

        rootDir := cfg.RootDir
        if len(rootDir) == 0 {
                rootDir = filepath.Join(hubRootDir, hn)
        }
```

But yes, I think it is okay to simply clear the default path, since nobody bothers to use different config XD